ui/src/lib/gql/trigger/useApi.ts

```diff
@@ -113,17 +114,23 @@ export const useTrigger = () => {
     }
   };
 
-  const useGetTriggersInfinite = (workspaceId?: string) => {
-    const { data, ...rest } = useGetTriggersInfiniteQuery(workspaceId);
+  const useGetTriggers = (
+    workspaceId?: string,
+    paginationOptions?: PaginationOptions,
+  ) => {
+    const { data, ...rest } = useGetTriggersQuery(
+      workspaceId,
+      paginationOptions,
+    );
     return {
-      pages: data?.pages,
+      pages: data,
```

Comment:
We aren't returning pages anymore, but (for now) a single page, so return either `page` or just `triggers`, `deployments`, etc. Please update all useApi files 🙏🏻